nuxeo-platform-spreadsheet-web/src/main/js/app/ui/spreadsheet.js

```diff
@@ -24,10 +25,9 @@ import {Query} from '../nuxeo/rpc/query';
  */
 class Spreadsheet {
 
-  constructor(container, connection, layout, columns, pageProvider, language) {
+  constructor(container, connection, resultLayoutName, resultColumns, pageProviderName, language) {
```

Comment:
IMHO the names better represent the values they are holding, and it confused because we already have this.columns, and layout vars in the same scope. But I've made the changes anyway.